sdk/storage/Azure.Storage.Files/src/Models/FileNtfsAttributes.cs

```diff
@@ -0,0 +1,285 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Azure.Storage.Files.Models;
+
+namespace Azure.Storage.Files.Models
+{
+    /// <summary>
+    /// NTFS file attributes for Files and Directories.
+    /// </summary>
+    public struct FileNtfsAttributes : IEquatable<FileNtfsAttributes>
```

Comment:
Why did you go with `FileNtfsAttributes` over `NtfsFileAttributes`?

Reply:
I just picked a name.  I'm open to renaming this.

Reply:
fixed.